AttitudeManager/FlightModes/Src/manual.cpp

```diff
@@ -3,10 +3,28 @@
 
 namespace AM {
 
-AttitudeManagerInput Manual::run(const AttitudeManagerInput& input) {
+AttitudeManagerInput Manual::run(const AttitudeManagerInput& input, AxisPIDs _pids) {
     return input;
 }
 
 void Manual::updatePid() {}
+void Manual::updatePidGains(PIDController _axis, GainTerm whichGain, float desiredGain) {
+    if (isnan(desiredGain)) {
+        return;
+    }
+
+    switch (whichGain) {
+        case proportional:
+            _axis.setKp(desiredGain);
+            break;
+        case integral:
+            _axis.setKi(desiredGain);
+            break;
+        case derivative:
+            _axis.setKd(desiredGain);
+            break;
```

Comment:
Remember to have a default case which throws some sort of error/ warning!

Also remember to do bounds checking on your whichGain :)